packages/web-components/fast-components/docs/api-report.md

```diff
@@ -20,10 +20,13 @@ import { Dialog } from '@microsoft/fast-foundation';
 import { Direction } from '@microsoft/fast-web-utilities';
 import { Divider } from '@microsoft/fast-foundation';
 import { Flipper } from '@microsoft/fast-foundation';
+import { Listbox } from '@microsoft/fast-foundation';
 import { Menu } from '@microsoft/fast-foundation';
 import { MenuItem } from '@microsoft/fast-foundation';
+import { Option as Option_2 } from '@microsoft/fast-foundation';
```

Comment:
`Option-2` seems out of place here.  How is this used?  Two without one or three?  Maybe "SubOption" or "MenuOption" would be more intuitive and align with how you've named other variables.

Reply:
yeah this seems wrong.

Reply:
I think this is happening because DOM already has an `Option()` constructor. Looking at the internals, this would likely happen with `Audio/<audio>` and `Image/<img>` as well.